doc/adoc/user_guide.adoc

```diff
@@ -273,7 +278,6 @@ a pointer to the respective log file.
   cloud framework is concerned, you will still be running an instance
   of the SLES version you started with. This cannot be changed.
 * The only supported migration path in the Public Cloud is from the
-  final 2 service packs of a distribution to the first service pack of
-  the next distribution. For example from SLES 12 SP4 or SLES 12 SP5 to
-  SLES 15 SP1. The packages delivered by SUSE in the Public Cloud Module
+  final 2 service packs of a distribution. For example from SLES 12 SP4 or SLES 12 SP5 to
+  SLES 15 SPX or . The packages delivered by SUSE in the Public Cloud Module
```

Comment:
This PR is an attempt to avoid ambiguity, IMHO it should state that the migration to 15 SP2 or 15 SP3 (or non default paths) happens when setting `migration_product` and not leave that to interpretation (reader could understand by default or sometimes, not being clear when)

my 2 cents

Reply:
I can call out one example in more detail based on Robert's feedback. But if none of these are "officially" supported we need to highlight that.  Something like

 "The default migration target  is SLES 15 SP1.  Using the migration_product the default path can be changed.  For example a migration_product set to ''SLES/15.3/x86_64' will change the target to SLES 15 SP3"

How does this sound?